vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -1420,6 +1421,8 @@ def fused_experts(
               w1, w2, inplace, activation, apply_router_weight_on_input,
               expert_map)):
         assert quant_config is not None
+        logger.info_once(
```

Comment:
Won't this cause issues with torch.compile logging during the forward pass? Have you validated this works?

Reply:
```bash
lm_eval   --model vllm   --model_args "pretrained=deepseek-ai/DeepSeek-V2-lite,max_model_len=32768,enable_expert_parallel=True"   --trust_remote_code   --tasks gsm8k   --num_fewshot 5   --batch_size auto

|Tasks|Version|     Filter     |n-shot|  Metric   |   |Value |   |Stderr|
|-----|------:|----------------|-----:|-----------|---|-----:|---|-----:|
|gsm8k|      3|flexible-extract|     5|exact_match|↑  |0.3700|±  |0.0133|
|     |       |strict-match    |     5|exact_match|↑  |0.3669|±  |0.0133|
```

I think the logging won't affect torch compile?

Reply:
Deleted the runtime log, just changing to debug and "available" now.